packages/manager/modules/pci/src/projects/project/notebooks/onboarding/onboarding.constants.js

```diff
@@ -0,0 +1,20 @@
+export const GUIDES = [
+  {
+    id: 'documentation',
+    link: 'https://docs.ovh.com/gb/en/public-cloud/notebooks/',
```

Comment:
can you support multiple language guides and use default gb guides? something like  https://github.com/ovh/manager/pull/4965/files#diff-02828e6144330239700b2b3ce1b5f63e6f8648fbec6a6a7f01ca7a60b17c4d68

Reply:
Not a part of this PR. But point noted. I will check with the PU on whether we will have multiple links...